python-sdk/src/astro/files/locations/base.py

```diff
@@ -72,14 +81,14 @@ def openlineage_dataset_name(self) -> str:
         raise NotImplementedError
 
     @staticmethod
-    def is_valid_path(path: str) -> bool:
+    def is_valid_path(path: str, conn_id: str | None = None) -> bool:
```

Comment:
I don't like this

Reply:
Removed this as this is optional. https://github.com/astronomer/astro-sdk/pull/1478/commits/8f3e88f213d3fa87c82ba21251b258848fa0dcc4